api/mapping/views.py

```diff
@@ -215,11 +216,15 @@ class ProjectListView(ListAPIView):
     filterset_fields = {"name": ["in", "exact"]}
 
     def get_serializer_class(self):
+        print(self.request.GET)
         if (
             self.request.GET.get("name") != None
             or self.request.GET.get("name__in") != None
         ):
             return ProjectSerializer
+        if self.request.GET.get("datasets") != None:
```

Comment:
same for the above lines too, I think PEP8 prefers you to test if something is `None` by doing `my_variable is not None` as opposed to `!= None`